internal/admission/defaulting_test.go

```diff
@@ -112,7 +112,7 @@ func TestTimeout(t *testing.T) {
 		require.NotNil(t, res.Result, "response is ok")
 		assert.Contains(t, res.Result.Message, "request exceeded desired timeout")
 		assert.True(t, res.Allowed)
-		assert.Equal(t, withAddRejectAnnotation(patchWithAddLabel(pkg.ValidationStatusPending)), res.Patches)
+		assert.EqualValues(t, withAddRejectAnnotation(patchWithAddLabel(pkg.ValidationStatusPending)), res.Patches)
```

Comment:
I used different correct method in next PR: https://github.com/kyma-project/warden/pull/80/files